src/Simplex/Chat.hs

```diff
@@ -525,7 +526,18 @@ processChatCommand = \case
         withStore' $ \db -> deleteGroupConnectionsAndFiles db user gInfo members
         withStore' $ \db -> deleteGroupItemsAndMembers db user gInfo members
         withStore' $ \db -> deleteGroup db user gInfo
+        forM_ contactIds $ \ctId ->
+          deleteUnusedContact ctId `catchError` \_ -> pure ()
```

Comment:
we have catchAll_ for that, but shouldn't we send these errors to view?

Reply:
it seems to do a different thing.. ; I don't think we should - it's mostly to workaround around the fact that `getContact` is failable and in this context it realistically can fail if a contact was deleted before group